setup.cfg

```diff
@@ -96,12 +92,8 @@ tracing =
     opentelemetry-exporter-zipkin
 
 [options.package_data]
-bentoml =
-    py.typed
-    bentoml/*
-bentoml_cli =
-    py.typed
-    bentoml_cli/*
+bentoml = bentoml/*
```

Comment:
let's put these type of changes to a separate PR and test separately next time @aarnphm